micro-metrics-datadog/src/main/java/com/aol/micro/server/datadog/metrics/DatadogMetricsConfigurer.java

```diff
@@ -32,18 +33,22 @@
     @Getter
     private EnumSet<DatadogReporter.Expansion> expansions;
 
+    private final String host;
+
     @Autowired
     public DatadogMetricsConfigurer(@Value("${datadog.apikey}") String apiKey,
```

Comment:
I don't think I do. This class is initialized as bean and default value is provided.